src/redux-toolkit/thunks/homePageThunk.ts

```diff
@@ -0,0 +1,37 @@
+import { createAsyncThunk } from "@reduxjs/toolkit";
+import axios from "axios";
+
+import { Section, ChefData } from "../../models/types";
+import { transformRestaurantData } from "../adapters/restaurantAdapter";
+import { transformDishData } from "../adapters/dishAdapter";
+import { transformChefData } from "../adapters/chefAdapter";
+
+interface HomePageData {
+  popularRestaurants: Section;
+  signatureDishes: Section;
+  chefOfTheWeek: ChefData;
+}
+
+export const fetchHomePageData = createAsyncThunk<HomePageData>(
+  "homePage/fetchData",
+  async (): Promise<HomePageData> => {
+    const [popularRestaurantsResponse, signatureDishesResponse, chefOfTheWeekResponse] = await Promise.all([
+      axios.get("http://localhost:3000/restaurants/popular"),
```

Comment:
Set a service for the api call for all kinds if api call using axios, and import from there